api/app/tests/Integration/Controller/DeputyControllerTest.php

```diff
@@ -190,8 +192,9 @@ public function testGetDeputyReportsReturnsResults()
         self::$fixtures->flush();
 
         // generate courtOrder and set client and deputy
-        $courtOrder = self::$fixtures->createCourtOrder(7055555550, 'pfa', true);
+        $courtOrder = self::$fixtures->createCourtOrder(7055555550, 'pfa', 'ACTIVE');
         $courtOrder->setClient($client);
+        $courtOrder->setStatus('ACTIVE');
```

Comment:
Is this line redundant, as you're setting the status when the court order is created?